### PR TITLE
Links on your website are VERY broken.

### PR DESCRIPTION
**Pull request by GuzioMG**
This is self-explaining. Just look at changes I made. And if (whoever is checking this pull) is to lazy, let me explain:

While I was looking for a decent video streaming library, I found webRtc. "Great!" - I thought. And I found a link, saying I need to figure something out first. Okay. I clicked it. And what I seen? Error404. Fortunatley, I decided to check URL and it turned out to be just a double backslash instead of singe one. "Easy fix"! ...Not really. Because in GitHub, it turned out to have a different folder structure. Therefore, no link worked there (yes, I know this sentence is not grammatically correct). So I decided to fix them. But it turned out to be just too long work. So... Yes. I ended up being tired and fixed just webRtc. In fact, I'm not even sure, is it actually working, because I had no way to check it. (I'm sure at least one link, close to the end isn't working correctly, but my brain was temporarily offline and I couldn't figure out, what's wrong.)

So to summarize, you still have to fix other links AND check those I fixed. Well, at lest I reported an error and started this work. "Beginning something is 50% of work to do." - said Guzio (me), who has billions of dead repos.

### CODE REVIEW

**Comment by yasserf** [comment]
Thanks for raising and the comedic effect.

Yeah the structure is different in the website because it indicates order. There isn't a concept of a github link anywhere within the actual content, not certain why you added decided it here tbh, sir.

I have personally stopped spending any time on this project other than responding to critical issues, mentioned in depth here (https://github.com/deepstreamIO/deepstream.io/issues/1030).

Edit: Last part was mostly around saving you time ;)

Also deepstream isn't a video streaming library nor is it really the recommended way to use webRTC 😅

**Comment by GuzioMG** [comment]
> Also deepstream isn't a video streaming library nor is it really the recommended way to use webRTC 😅

I noticed, but whatever. Sufficient for what I'm planning to do.

**Comment by GuzioMG** [comment]
> responding to critical issues

While trying to make every link github-friendly might not be the best idea, I'd consider fixing at least the backshlash problem, because it might be a bit of a roadblock (-ish? thing???) for some users. And it's critical then. Kind of. At least... It's kinda important.

**Comment by GuzioMG** [comment]
Anyway, I'm closing this nonsense. Backshlashes can be fixed without my help and that "GitHub friendly link: " stuff is just causing chaos here. I'll also remove my fork, but here you have a bunch of places, where I found double backshlashes:

content/tutorials/98-webrtc/30-webrtc-audio-video/index.md (line 7)
content/tutorials/98-webrtc/60-webrtc-file-transfer/index.md  (line 7)
 content/tutorials/98-webrtc/10-webrtc-datachannels/index.md (line 25)

And there are more definitley!!!

**Comment by GuzioMG** [comment]
Stupid GitHub. It said "close and comment" and did it in the opposite direction.

Anyway, bye and thanks for your attention!
